### DISCUSSION TITLE
How to find all parameters contain System.Byte[]?

### DISCUSSION THREAD

**Discussion by usbpendrive** [vfsfitvnm/frida-il2cpp-bridge#86] [Q&A]
Hi,
How to find parameters contain System.Byte[] (byte array)?

**Comment by vfsfitvnm** [comment]
Hi, I don't really get what you are trying to achieve. Could you give further details?

**Comment by usbpendrive** [comment]
I'm trying to find all parameters that contains System.Byte[] (byte array). How can I achieved that, using Il2cpp.Array doesn't seem to do the job. How to create a System.Byte[] as parameter.

**Comment by vfsfitvnm** [comment]
You can dump the classes first (https://github.com/vfsfitvnm/frida-il2cpp-bridge/wiki/Snippets#dumping), then look for `System.Byte[]` with a text viewer. I personally use `klogg`
